packages/core/src/pagination/pagination-button.element.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { baseStyles, CdsBaseButton, property, querySlot } from '@cds/core/internal';
+import { html } from 'lit-element';
+import { styles } from './pagination-button.element.css.js';
+
+export type CdsPaginationButtonAction = 'prev' | 'next' | 'first' | 'last';
```

Comment:
If `CdsPaginationButtonAction` is only being used in this file, we probably don't need to export it.